org-super-agenda.el

```diff
@@ -1099,6 +1114,53 @@ actually the ORDER for the groups."
        (-remove #'s-blank-str? it)
        (s-join "\n" it)))
 
+;;;; Hide/Show filter
+(defun org-super-agenda--hide-or-show-groups (&rest _)
+  "Hide/Show any empty/non-empty groups after `org-agenda-finalize',
+`org-agenda-filter-apply'  was called."
+  (cl-labels ((header-p () (org-get-at-bol 'org-super-agenda-header))
+              (grid-p () (not (or  (org-get-at-bol 'org-agenda-structural-header)
+                                   (org-get-at-bol 'org-agenda-date-header)
+                                   (org-get-at-bol 'type)
+                                   (org-get-at-bol 'org-super-agenda-header))))
+              (group-item-visible-p () (and (org-get-at-bol 'type) (not (org-get-at-bol 'invisible))))
+              (next-header () (let (header nohide grid-end)
+                                (while (not (or (bobp) header))
+                                  (cond
+                                    ((header-p)
+                                     (setq header (list (1- (or (previous-single-property-change (point-at-eol) 'org-super-agenda-header) (1+ (point-min))))
+                                                        (or grid-end (point-at-eol))
+                                                        nohide)))
+                                    ((group-item-visible-p)
+                                     (setq nohide t))
```

Comment:
Another nitpick: would it make the code clearer if it were `hide-p` rather than `nohide`?  Of course that would invert its meaning, but it might be worth it.  What do you think?

Reply:
Man, I thought I'm fussy :-D. If I show that my working colleagues they hopefully change there mind about me.

Reply:
A wise man once said, "If you take care of the little things, the big things will take care of themselves."  ;)